github/scim.go

```diff
@@ -59,20 +59,46 @@ type ListSCIMProvisionedIdentitiesOptions struct {
 	Filter *string `json:"filter,omitempty"`
 }
 
+// SCIMUser - GitHub refers to the SCIM schema here: https://datatracker.ietf.org/doc/html/rfc7643#section-4.1
+// it's possible that the exact schema (and applicable extensions) to be used should be determined by reading the
+// schemas field and that this should therefore not be hard coded to use only the currently used schema?
+type SCIMUser struct {
+	Id   *string  `json:"id,omitempty"`
+	Meta SCIMMeta `json:"meta,omitempty"`
+	SCIMUserAttributes
+}
+
+type SCIMMeta struct {
```

Comment:
Please add a godoc comment in the form of a complete sentence (with the trailing period).
For example, `// SCIMMeta represents ... blah blah.`